docs/TpchBenchmark.md

```diff
@@ -0,0 +1,453 @@
+# Velox I/O Optimizations and the TpchBenchmark
+## Introduction
+This document is outcome from a cycle of benchmarking to determine best I/O performance against AWS S3 Parquet data for TPCH. It is intended to describe tuning process recommendations and general suggestions for tuning the Velox query engine.
+
+Benchmarking in Velox is made easy with the optionally built TpchBenchmark (velox_tpch_benchmark) executable. To build the benchmark executable _(build/release/velox/benchamrks/tpch/velox_tpch_benchmark)_, patch the code base with a small patch ([Appendix A](#appendix-a-patch-to-build-tpchbenchmark-executable)) then use the following command line to do the build:
```

Comment:
Curious - why the need to patch the code ? Is the code being patched in our dependencies ?

Reply:
Will resolve the document as if this is fixed and we can delay merging until after the future PR (https://github.com/facebookincubator/velox/pull/5366) is merged.